hazelcast/src/main/java/com/hazelcast/config/CacheConfig.java

```diff
@@ -75,7 +75,7 @@
      * Disables invalidation events for per entry but full-flush invalidation events are still enabled.
      * Full-flush invalidation means the invalidation of events for all entries when clear is called.
      */
-    private boolean disablePerEntryInvalidationEvents;
+    private boolean disablePerEntryInvalidationEvents = true;
```

Comment:
Isn't this changing the default `Cache` configuration? What's the reasoning for this change?